Packs/Cyberint/Integrations/Cyberint/Cyberint.py

```diff
@@ -559,9 +627,11 @@ def main():
             fetch_type = params.get('fetch_type', [])
             fetch_severity = params.get('fetch_severity', [])
             max_fetch = int(params.get('max_fetch', '50'))
-            next_run, incidents = fetch_incidents(
-                client, demisto.getLastRun(), first_fetch_time, fetch_severity, fetch_status,
-                fetch_type, fetch_environment, max_fetch)
+            fetch_assign_multiple = params.get('assign_multiple', True)

```

Comment:
Consider renaming the variable as displayed in the previous comments. In addition, are we sure that the default value is multiple - which means we duplicate the alerts as the default use case?